content/blog/2016/2016-05-26-pipeline-snippetizer.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: New display of Pipeline’s “snippet generator”
+tags:
+- pipeline
+author: jglick
+---
+
+Those of you updating the https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Groovy+Plugin[Pipeline Groovy plugin]
+to 2.3 or later will notice a change to the appearance of the configuration form.
+The _Snippet Generator_ tool is no longer a checkbox enabled inside the configuration page.
+Rather, there is a link _Pipeline Syntax_ which opens a separate page with several options.
+(The link appears in the project’s sidebar; Jenkins 2 users will not see the sidebar from the configuration screen,
+so as of 2.4 there is also a link beneath the Pipeline definition.)
+
+_Snippet Generator_ continues to be available for learning the available Pipeline steps
+and creating sample calls given various configuration options.
+The new page also offers clearer links to static reference documentation, online Pipeline documentation resources,
+and an IntelliJ IDEA code completion file (Eclipse support is unfinished).
+
+One motivation for this change (https://issues.jenkins-ci.org/browse/JENKINS-31831[JENKINS-31831])
+was to give these resources more visual space and more prominence.
+But another consideration was that people using multibranch projects or organization folders
+should be able to use _Snippet Generator_ when setting up the project, before any code is committed.
+To that end, also update the https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Multibranch+Plugin[Pipeline Multibranch plugin]
```

Comment:
> To that end, also update the Pipeline Multibranch plugin to 2.4 or later, to get the same link in these contexts

This is only a suggestion, but I find this sentence really confusing to parse (I had to read it twice). 

Perhaps:

> Those using Pipeline Multibranch or organization folder plugins should upgrade to 2.4 or later to see these improvements as well.